test/suite/test_lsm02.py

```diff
@@ -59,22 +59,20 @@ def test_lsm_tombstone(self):
         self.add_key(self.uri, 'k3', v)
         self.verify_key_exists(self.uri, 'k3', v)
 
-    def test_lsm_rename01(self):
+    def test_lsm_create01(self):
```

Comment:
I think this whole function can be dropped - the test is now just adding a key then verifying it exists.